spec/person_spec.rb

```diff
@@ -0,0 +1,8 @@
+require './person'
+describe Person do
+  let(:person) { Person.new(31, 'Mumbere Habert') }
```

Comment:
- kindly test the situation were the constructor method sets a default value for the optional name parameter if the name is not provided by the user. Hints: expect(person.name).to eql "Unknown" as previous code reviwer seggested👍